### PR TITLE
Data Abdulwahid7027

### PR DESCRIPTION
**Pull request by Abdulwahid7027**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2873"></a>

### CODE REVIEW

File: data/Abdulwahid7027.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Abdulwahid702",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "www.linkedin.com/in/abdul-wahid-shaik-611893259/",
```
Comment (Aadarsh805) [inline data/Abdulwahid7027.json:33]: fix your LinkedIn URL, it should be `https://www.linkedin.com/in/abdul-wahid-shaik-611893259/`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/Abdulwahid7027.json (hunk 1 above)
Comment (Abdulwahid7027) [reply data/Abdulwahid7027.json]: ## Ok

File: data/Abdulwahid7027.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Shaik Abdul Wahid",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student • Frontend developer • Open Source Enthusiast • Python • Java",
+  "avatar": "https://avatars.githubusercontent.com/u/110891341?s=96&v=4",
```
Comment (senali-d) [inline data/Abdulwahid7027.json:6]: Update the avatar url to short link.
```suggestion
  "avatar": "https://github.com/Abdulwahid7027.png",
```

**Review by senali-d — CHANGES_REQUESTED** [review]

**Review by Abdulwahid7027 — COMMENTED** [review]
## Thanks

**Comment by senali-d** [comment]
LGTM 👍

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**